src/eva/vision/models/networks/decoders/segmentation/linear.py

```diff
@@ -30,6 +30,14 @@ def __init__(self, layers: nn.Module) -> None:
     def _forward_features(self, features: List[torch.Tensor]) -> torch.Tensor:
         """Forward function for multi-level feature maps to a single one.
 
+        It will interpolate the features and concat them into a single tensor
+        on the dimension axis of the hidden size.
+
+        Example:
+            features = [torch.Size([16, 384, 14, 14]), torch.Size([16, 384, 14, 14])]
+            output = self._forward_features(features)
+            assert output.shape == torch.Size([16, 768, 14, 14])
```

Comment:
adapt to conform with docstring style guide? https://kaiko-ai.github.io/eva/latest/STYLE_GUIDE/?h=example#docstrings

Reply:
nice done